src/components/Groups/JoinGroup/JoinGroupForm.tsx

```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import { Button, ButtonText } from '../../basic';
 import { Container } from './join_group_styles';
 import Picker from './Picker';
+import { StyleSheet } from 'react-native'
 
 export const JoinGroupForm = props => (
     <Container>
         <Picker {...props} />
-        <Button onPress={props.handleSubmit} disabled={!props.isValid}>
+        <Button onPress={props.handleSubmit} disabled={!props.isValid} style={styles.JoinGroupButtonStyle}>
```

Comment:
Same with this remove the style prop and pass in `backgroundColor` prop which you can access in the button.